<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyTcpServerWidget</class>
 <widget class="QWidget" name="MyTcpServerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyTcpServerWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QTextEdit" name="receiveTextBox">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QTextEdit" name="sendTextBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="sendBtn">
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>452</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="closeBtn">
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
